### ISSUE TITLE
Evaluation coordination

### ISSUE THREAD

**Issue by dgedon** [antonior92/physionet-12ecg-classification#16]
In order to coordinate the evaluation for the first submission, we structure the evaluation a bit clearer.

Take the following abbreviations of a specific sub-model:
1. Pre- training Stage (default = Transformer, 1.RNN = RNN)
2. Model (default = ResNet, 2.MLP = MLP)
3. Prediction Stage (default = RNN, 3.Mean = linear+mean, 3.Max = linear+max)
4. Last Layer (default = hierarchical, 4.Sig/Soft = sigmoid/softmax)

We want the following for our first submission:
- **Transformer** (we compare with MLP instead of ResNet due to size of Transformer here): 
- [x] 1+2.MLP > 1.RNN+2.MLP  (it needs more work!)
- **Model**: 
- [ ] 1 +2 > 1+2.MLP
- [ ] 1+2 > 2 
- **Prediction Stage**: 
- [ ] 1+2+3 > 1+2+3.Mean
- [ ]  1+2+3 > 1+2+3.Max
- **Last Layer**: 
- [x] 1+2+3+4 > 1+2+3+4.Sig/Soft
- [ ] Combinations of softmax

Please tick if you have a model that confirms one of the evaluations and safe the hyperparameters and options for this.
